### PR TITLE
feat/performance boots

### PR DESCRIPTION
**Pull request by WilsonNet**
# Description

This PR remove the precounts of the status for each builds and put it under demand when the user opens an accordion

You can test the graphs in this link:
http://localhost:5173/tree/034835342aab1cc7dc1733a98cd2a6d159ee242a?tableFilter=%7B%22bootsTable%22%3A%22all%22%2C%22buildsTable%22%3A%22all%22%2C%22testsTable%22%3A%22all%22%7D&origin=maestro&currentTreeDetailsTab=treeDetails.builds&diffFilter=%7B%7D&treeInfo=%7B%22gitUrl%22%3A%22https%3A%2F%2Fgit.kernel.org%2Fpub%2Fscm%2Flinux%2Fkernel%2Fgit%2Famlogic%2Flinux.git%22%2C%22gitBranch%22%3A%22for-next%22%2C%22treeName%22%3A%22amlogic%22%2C%22commitName%22%3A%22v6.11-rc1-22-g034835342aab%22%2C%22headCommitHash%22%3A%22034835342aab1cc7dc1733a98cd2a6d159ee242a%22%7D

And the slow repo in this link:

http://localhost:5173/tree/fe3b83691989077c55cd1e91e0a0bd831b6255aa?tableFilter=%7B%22bootsTable%22%3A%22all%22%2C%22buildsTable%22%3A%22all%22%2C%22testsTable%22%3A%22all%22%7D&origin=maestro&currentTreeDetailsTab=treeDetails.builds&diffFilter=%7B%7D&treeInfo=%7B%22gitUrl%22%3A%22https%3A%2F%2Fgit.kernel.org%2Fpub%2Fscm%2Flinux%2Fkernel%2Fgit%2Fnext%2Flinux-next.git%22%2C%22gitBranch%22%3A%22pending-fixes%22%2C%22treeName%22%3A%22next%22%2C%22commitName%22%3A%22v6.11-rc7-98-gfe3b83691989%22%2C%22headCommitHash%22%3A%22fe3b83691989077c55cd1e91e0a0bd831b6255aa%22%7D

### CODE REVIEW

File: backend/kernelCI_app/views/treeDetailsView.py [hunk 1]
```diff
@@ -11,95 +11,86 @@ def create_default_status(self):
...
+                query.where(**{f"{table}.{field}__{op}": f["value"]})
 
         records = query.select()
-        for r in records:
-            status = self.get_test_status(r.get('id'))
-            r['status'] = status
 
         summary = self.create_summary(records)
 
+        print("Summary: ", summary)
```
Comment (lfjnascimento) [inline backend/kernelCI_app/views/treeDetailsView.py:94]: leftover

File: dashboard/src/components/Accordion/BuildAccordionContent.tsx [hunk 2]
```diff
@@ -45,65 +52,95 @@ export interface ILinksGroup {
...
+
+  console.log('oii', accordionData);
+
+  //TODO: Fix the typing for not using as
+  const contentData = accordionData as AccordionItemBuilds;
+
+  const { data, status } = useBuildStatusCount(
+    { buildId: contentData.id ?? '' },
+    { enabled: !!contentData.id },
+  );
```
Comment (lfjnascimento) [inline dashboard/src/components/Accordion/BuildAccordionContent.tsx:132]: `useBuildStatusCount` can receive only id and "calculate" `enabled` if

File: dashboard/src/components/Accordion/BuildAccordionContent.tsx [hunk 3]
```diff
@@ -45,65 +52,95 @@ export interface ILinksGroup {
...
+    </>
+  );
+};
+
+const AccordionBuildContent = ({
+  accordionData,
+}: IAccordionItems): JSX.Element => {
+  const { treeId } = useParams({ from: '/tree/$treeId/' });
+
+  console.log('oii', accordionData);
```
Comment (lfjnascimento) [inline dashboard/src/components/Accordion/BuildAccordionContent.tsx:124]: leftover

File: backend/kernelCI_app/views/buildStatusCountView.py [hunk 4]
```diff
@@ -0,0 +1,48 @@
...
+        build_counts = {
+            "build_id": build_status.id,
+            "fail_tests": build_status.fail_tests,
+            "error_tests": build_status.error_tests,
+            "miss_tests": build_status.miss_tests,
+            "pass_tests": build_status.pass_tests,
+            "done_tests": build_status.done_tests,
+            "skip_tests": build_status.skip_tests,
+            "null_tests": build_status.null_tests,
+            "total_tests": build_status.total_tests,
```
Comment (lfjnascimento) [inline backend/kernelCI_app/views/buildStatusCountView.py:45]: I would say that we should create a pattern and always use the same names for status (maybe use the same as KCIDB "MISS, PASS, DONE") in all our queries, but it's up to you whether to address this suggestion for this query now or latter.

Comment (WilsonNet) [reply backend/kernelCI_app/views/buildStatusCountView.py:45]: we can do that but we need to have some way to standardize beside our mouths, probably pydantic will serve for that purpose

**Pull request merged**